src/Assetic/Filter/LessFilter.php

```diff
@@ -51,20 +57,29 @@ class LessFilter extends BaseNodeFilter implements DependencyExtractorInterface
      * @param string $nodeBin   The path to the node binary
      * @param array  $nodePaths An array of node paths
      */
-    public function __construct($nodeBin = '/usr/bin/node', array $nodePaths = array())
+    public function __construct()
```

Comment:
Tthis doesn't break BC as those arguments will ignored by the constructor if provided.